transformations/normalize_to_ascii/requirements.txt

```diff
@@ -0,0 +1 @@
+text-unidecode==1.3
```

Comment:
Hi @PhilipEHausner, thank you for your efforts. This is GPL licensed - please use either an MIT or Apache 2 Licensed.

Reply:
Hello @kaustubhdhole and thank you for the quick reply. I had a look at https://pypi.org/project/text-unidecode/ which states that the package can be either distributed under the artistic license or GPL, even stating that if GPL licensing is OK within a project, unidecode (instead of text-unidecode) can be used. Since I assumed (after some research) that the artistic license is a permissive license, I assumed it would be also in line with the restrictions for NL-Augmenter. Did I miss something here, or do you consider the artistic license to not be permissive?

Reply:
Hi @PhilipEHausner, apologies for the delayed reply. The artistic license has restrictions and we would suggest you to use other licensed packages - especially MIT or Apache 2.